### DISCUSSION TITLE
Problem while running inference code

### DISCUSSION THREAD

**Discussion by 250519** [OpenTalker/SadTalker#953] [Q&A]
<img width="682" alt="Screenshot 2024-08-19 at 1 20 48 AM" src="https://github.com/user-attachments/assets/25448e99-0dbe-414f-aa61-9b3d8c161847">
It is not being processes further may I know whats the issue?

**Comment by samarasimhapeyala** [comment]
Encountered same issues, it's automatically killed, in my case it was due to the Colab resources are not enough in that step. Tried on the kaggle notebooks, it worked.
